packages/components/src/functional-components/inputs/NativeOptionList/NativeOptionList.tsx

```diff
@@ -0,0 +1,50 @@
+import { h, Fragment, type FunctionalComponent as FC } from '@stencil/core';
+import type { SelectOption, W3CInputValue } from '../../../schema';
+import NativeOptionFc from '../NativeOption/NativeOption';
+import type { JSXBase } from '@stencil/core/internal';
+
+export type NativeOptionListProps = {
+	preKey?: string;
+	disabled?: boolean;
+	value?: W3CInputValue | W3CInputValue[];
+	options?: SelectOption<W3CInputValue>[];
+
+	OptionProps?: Omit<JSXBase.OptionHTMLAttributes<HTMLOptionElement>, 'value' | 'label'>;
+	OptionGroupProps?: Omit<JSXBase.OptgroupHTMLAttributes<HTMLOptGroupElement>, 'label'>;
+};
+
+const NativeOptionListFc: FC<NativeOptionListProps> = ({ preKey, options, value: selectedValue, OptionProps = {}, OptionGroupProps = {} }) => {
+	if (!options?.length) {
+		return null;
+	}
+
+	return (
+		<>
+			{options.map((option, index) => {
+				const key = [preKey, `-${index}`].join('');
```

Comment:
```suggestion
				const key = `${preKey}-${index}`;
```
Würde ich persönlich vereinfachen, bin beim Lesen über die Kombination aus join und Template-Strings gestolpert.

Reply:
Mein Variante ist halt undefined sicher, für den preKey